custom/icds_reports/tasks.py

```diff
@@ -953,9 +953,9 @@ def create_mbt_for_month(state_id, month):
     helpers = (CcsMbtHelper, ChildHealthMbtHelper, AwcMbtHelper)
     for helper_class in helpers:
         helper = helper_class(state_id, month)
-        with get_cursor(helper.base_class) as cursor:
-            cursor.execute(helper.query())
-        with open(helper.output_file, 'r') as f:
+        with get_cursor(helper.base_class) as cursor, open(helper.output_file, 'r+') as f:
+            cursor.copy_expert(helper.query(), f)
+            f.seek(0)
```

Comment:
Does having this inside the same context manager as get_cursor leave a transaction open for a long time?

If it does it may be better to have nested managers:

```python
with open() as f:
    with get_cursor() as cursor:
        cursor.copy_expert()
    # do stuff with file
```